shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate.mm

```diff
@@ -248,6 +248,18 @@ - (void)application:(UIApplication*)application
   }
 }
 
+- (void)application:(UIApplication*)application
+    didFailToRegisterForRemoteNotificationsWithError:(NSError*)error {
+  for (NSObject<FlutterApplicationLifeCycleDelegate>* delegate in _delegates) {
+    if (!delegate) {
```

Comment:
```objc
if (delegate == nil) {... }
```

Reply:
I'm just following the style of the rest of the file. I don't think there is a difference between them, do you?